src/blight/actions/find_inputs.py

```diff
@@ -89,15 +89,16 @@ def before_run(self, tool: Tool) -> None:
 
             kind = INPUT_SUFFIX_KIND_MAP.get(input_path.suffix, InputKind.Unknown)
 
-            inputs.append(Input(prenormalized_path=input, kind=kind, path=input_path))
+            inputs.append(Input(prenormalized_path=input, kind=kind, path=str(input_path)))
 
         self._inputs = inputs
 
     def after_run(self, tool: Tool, *, run_skipped: bool = False) -> None:
         inputs = InputsRecord(tool=tool, inputs=self._inputs)
 
         if tool.is_journaling():
-            self._result = inputs.dict()
+            # NOTE(ms): The `tool` member is excluded to avoid journal bloat.
+            self._result = inputs.dict(exclude={"tool"})
```

Comment:
Nitpick: Instead of `exclude=...`, we could make `inputs.tool` into `inputs._tool` and Pydantic will automatically exclude it.

Reply:
Actually, ignore that. I see you're excluding it conditionally.